bedrock/products/urls.py

```diff
@@ -49,6 +49,14 @@
         ),
         name="products.vpn.resource-center.article",
     ),
+    # Monitor
+    path(
+        "monitor/",
+        prefer_cms(
+            views.resource_center_landing_view,
```

Comment:
So if there's no CMS version of the monitor page, it'll try to load the VPN Resource Centre index page.

What might be better is to not have this and - in order to be able to use `reverse()` in Python or the `url()` helper in templates - you add an equivalent route to the `cms_only_urls.py` file. The only reason to use `prefer_cms` here is if there's a legacy Monitor page we want to fall back to, which I don't think there is, yeap?